server/app/services/geo/esri/RealEsriClient.java

```diff
@@ -81,21 +82,40 @@ public RealEsriClient(
     checkNotNull(settingsManifest);
     this.ws = checkNotNull(ws);
 
+    this.ESRI_EXTERNAL_CALL_TRIES = settingsManifest.getEsriExternalCallTries().orElse(3);
+    this.ESRI_WELLKNOWN_ID_OVERRIDE = settingsManifest.getEsriWellknownIdOverride();
+    this.ESRI_ARCGIS_API_TOKEN = settingsManifest.getEsriArcgisApiToken();
+    this.ESRI_FIND_ADDRESS_CANDIDATES_URLS = getFindAddressCandidateUrls(settingsManifest);
+  }
+
+  /** Get the list of urls that will be used to correct addresses */
+  private static ImmutableList<String> getFindAddressCandidateUrls(
```

Comment:
Originally I was going to just make it `@VisibleForTesting`, but since `ESRI_FIND_ADDRESS_CANDIDATES_URLS` already is visible it I went with that. I don't like exposing things just for testing anyways so win/win.

I leave it static here since it doesn't need to rely on any class level variables/state.

Reply:
I was just thinking that if it was an instance method, you could use the `ESRI_ARCGIS_API_TOKEN` rather recreating it as a local variable, but I see what you mean about not relying on class level state.